[YUNIKORN-1788] Fix UI styling after Angular 16 update

craigcondit opened the pull request:
### What is this PR for?
Fixes the UI glitches that appeared after the Angular 16 update recently.

### What type of PR is it?
* [x] - Bug Fix
* [ ] - Improvement
* [ ] - Feature
* [ ] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-1788

### How should this be tested?

### Screenshots (if appropriate)
<img width="1699" alt="dashboard" src="https://github.com/apache/yunikorn-web/assets/12699633/736d6e38-f786-4e52-b06f-0de761be420a">
<img width="1702" alt="queues" src="https://github.com/apache/yunikorn-web/assets/12699633/10fa5db7-a41c-4e71-80cd-05152b0fd6a5">
<img width="1706" alt="apps" src="https://github.com/apache/yunikorn-web/assets/12699633/ad2b25c5-1403-4d19-9d13-69d299d57db6">
<img width="1700" alt="nodes" src="https://github.com/apache/yunikorn-web/assets/12699633/ef2f5886-b440-4aeb-9866-343c22d5d4c0">

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

wilfred-s commented on src/app/app.component.scss:
```diff
@@ -247,6 +247,7 @@ ul.navigation-primary {
 
 .help-menu > .menu-trigger {
   background: transparent;
+  font-size: 14px;
```
This should be the same change as in the .info-card > .menu-trigger` of the dashboard so the **?** and **i** use the same formatting.

craigcondit replied:
Done

wilfred-s reviewed (changes requested):
Angular 15 mentions it in the release notes that a tool has to be run it shows slightly different changes than you have made. I did not link that back to the objects we used while migrating. I found it based on your mdc changes:
https://material.angular.io/guide/mdc-migration
We need to run that tool as the base change, apply the healthcheck, about-menu, about-card and help-menu changes on top of that.

craigcondit commented:
> Angular 15 mentions it in the release notes that a tool has to be run it shows slightly different changes than you have made. I did not link that back to the objects we used while migrating. I found it based on your mdc changes: https://material.angular.io/guide/mdc-migration We need to run that tool as the base change, apply the healthcheck, about-menu, about-card and help-menu changes on top of that.

Good catch. Didn't realize there was an automated tool for this. I've updated the PR, also fixed a minor issue with the search box on the apps page where height was wrong and size was way too large to fit on a reasonable-sized window.

wilfred-s reviewed (approved):
LGTM
